docs/source/guides/data-scientist/00-connect-to-domain.rst

```diff
@@ -0,0 +1,170 @@
+      password="supersecurepassword",
+      url="localhost",
+      port=8081
+   )
+
+.. note::
+   By default, the role assigned to the registered user is of a Data Scientist, and the assigned privacy budget is 0.
+
+
+Step 2: Login to your Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have your login credentials, you can ask your Domain Owner to provide you with the link to the ``URL`` to access your Domain.
+
+.. WARNING:: 
+   If you do not have a domain owner, you can ``create`` one locally following the tutorials 
+   starting here: `Deploy your own Domain Server <../data-owner/00-deploy-domain.html>`_.
+
+Using the link, you can visit the Data Scientists ``login page``, as shown in the image below.
+
+|00-connect-to-domain-01|
+
+The ``Email`` and ``Password`` are your login credentials which you already have if you executed :ref:`Step 1 <step-ds-1>` above.
+
+Once we're ``logged`` in, you can move to the next section, which ``explores`` the user homepage.
+
+
+Step 3: Explore the User HomePage
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you login, the first page seen by the Data Scientists is the ``User`` homepage.
+
+When on this page, you can ``create`` an account for other team members within your organization and 
+check the requests for ``pending`` users and ``denied`` users. Also, it gives you information like privacy budget 
+allocated, date added, email, and the name of other users in a table-like format.
+
+|00-connect-to-domain-02|
+
+In the next section, you can set ``permissions`` for a user by their assigned roles. There are different ``roles`` 
+and their associated permissions, but you are restricted to managing the Domain as a user. You can ``review`` and 
+``customize`` what permissions apply to the roles on this page. 
+
+|00-connect-to-domain-03|
+
+``Data requests`` are one-time requests made by Data Scientists on the organization's Domain Node 
+to ``download`` the results of their computations. Unlike setting privacy budgets, data requests must 
+be manually triaged and do not count as ongoing credits. They are individual ``allowances`` based off 
+of specific computations on specified ``data objects``.
+
+On this page, you can check the ``pending`` data requests in your queue and the ``status`` of users' past data requests.
+
+|00-connect-to-domain-04|
+
+``Networks`` are the primary way to ``connect`` Data Scientists to any organization's Domain Node. Users 
+can join any public network and find the type of ``datasets`` they want to work on. They can search and 
+approve bulk project requests to simultaneously participate in a project across groups of domains. 
+
+Once the user agrees to the ``Network Agreements``, they can ``explore`` and ``analyze`` the ``datasets`` hosted 
+by the Domain Owners. In the image below are the three public domains that Data Scientists can join as 
+a ``guest``; alternatively, they can search for their preferred Domain using the search bar.
+
+|00-connect-to-domain-05|
```

Comment:
Instead, we can add a section for Exploration using the Python Syft Client. It doesn't need to dive deep but a quick give overview with some starting commands.


e.g. 
```
import syft as sy
domain_client = sy.login()

# name of the domain
domain_client.name

# View datasets on the domain
domain_client.datasets


# View store on the domain
domain_client.store
```
and then we can link it to the next tutorial, saying for more detail on data exploring refer to <Tutorial Name>